modules/uptime.sh

```diff
@@ -0,0 +1,31 @@
+function uptime {
+  checkrpi
+  checkargn $# 1
+  if [[ $1 == "" ]]; then
+    if ! dpkg-query -W -f='${Status}' uptimed | grep -q "ok installed";
+    then
+        sudo apt-get install uptimed
```

Comment:
You might want to move this package installation to builder

https://github.com/treehouses/builder/blob/23ad006b40d8e2365bce1d61c92e4a4f75411ead/scripts.d/16_pkg_install.sh#L5